### PR TITLE
feat: 🎸 Add auto-disconnect url on unauthenticated

### PR DESCRIPTION
**Pull request by KazerKrin**
#### Technical checklist

- [x] I tested my code following testing cases.
- [x] I covered my code with automatic tests.
- [x] I added logs in case of error or business logic.
- [x] I well designed my commits.
- [x] I read my Pull Request's changes.
- [x] I respected provided design and wording.

#### Pull request's scope

- [x] Ticket JIRA: [EPSOR-1411](https://projet-epsor.atlassian.net/browse/EPSOR-1411)
- [x] Pull Request's link added as comment to JIRA ticket.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -86,7 +86,10 @@ export default (
         if (graphQLErrors) {
           graphQLErrors.forEach(error => {
             if (error.extensions.code === 'UNAUTHENTICATED') {
-              window.location.replace(process.env.REACT_APP_AUTH_INTERFACE_FQDN);
+              const path = window.location.pathname.match(
+                /^https?:\/\/[a-zA-Z0-9-]*\.?${process.env.ENVIRONMENT}\.epsor\.fr$/,
+              );
+              window.location.replace(`${path}/deconnexion`);
```
Comment (dachristo) [inline src/index.js]:
```suggestion
              window.location.replace(`/deconnexion`);
```

File: src/index.js [hunk 2]
```diff
@@ -86,7 +86,10 @@ export default (
         if (graphQLErrors) {
           graphQLErrors.forEach(error => {
             if (error.extensions.code === 'UNAUTHENTICATED') {
-              window.location.replace(process.env.REACT_APP_AUTH_INTERFACE_FQDN);
+              const path = window.location.pathname.match(
+                /^https?:\/\/[a-zA-Z0-9-]*\.?${process.env.ENVIRONMENT}\.epsor\.fr$/,
+              );
```
Comment (dachristo) [inline src/index.js]:
```suggestion

```

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@epsor/apollo-client-wrapper",
-  "version": "2.0.13",
+  "version": "2.0.14",
```
Comment (Pangoraw) [inline package.json]: You should update `package-lock.json` as well.
You can do:
```
npm version patch
```
```
git reset HEAD^
```
```
npm run commit
```

File: src/index.js [hunk 4]
```diff
@@ -86,7 +86,7 @@ export default (
         if (graphQLErrors) {
           graphQLErrors.forEach(error => {
             if (error.extensions.code === 'UNAUTHENTICATED') {
-              window.location.replace(process.env.REACT_APP_AUTH_INTERFACE_FQDN);
+              window.location.replace('/deconnexion');
```
Comment (ppinel) [inline src/index.js]: No no no !
You can pass as params the logout url.

**Pull request merged**